presto/dbapi.py

```diff
@@ -41,6 +46,93 @@
 logger = presto.logging.get_logger(__name__)
 
 
+class ParamEscaper(object):
+    """
+    Escapes parameters for an SQL query in Presto. The entry point is the public method escape_args.
+    """
+
+    def escape_args(self, parameters):
```

Comment:
Could you please add a comprehensive unit test for this one, so we are sure all the corner cases are exercised?